File: Iniciante/C/1012.c
#include <stdio.h>

int main() {

    double A, B, C;
    double AreaTria, AreaCirc, AreaTrap, AreaQuad, AreaRetang;

    scanf("%lf %lf %lf", &A, &B, &C);

    AreaTria = (A*C) / 2;
    AreaCirc = 3.14159 * (C * C);
    AreaTrap = ( (A+B) * C )  / 2;
    AreaQuad = B * B;
    AreaRetang = A * B;

    printf("TRIANGULO: %0.3lf\n", AreaTria);
    printf("CIRCULO: %0.3lf\n", AreaCirc);
    printf("TRAPEZIO: %0.3lf\n", AreaTrap);
    printf("QUADRADO: %0.3lf\n", AreaQuad);
    printf("RETANGULO: %0.3lf\n", AreaRetang);

    return 0;
}

File: Iniciante/Python/1012.py
#beecrowd | 1012 - Área

dadosTriangulo = input().split()
ladoA = float(dadosTriangulo[0])
ladoB = float(dadosTriangulo[1])
ladoC = float(dadosTriangulo[2])

print("TRIANGULO: {:.3f}".format((ladoA * ladoC)/2))
print("CIRCULO: {:.3f}".format(3.14159 * (ladoC**2)))
print("TRAPEZIO: {:.3f}".format(((ladoA + ladoB) * ladoC) / 2))
print("QUADRADO: {:.3f}".format(ladoB**2))
print("RETANGULO: {:.3f}".format(ladoA * ladoB))
